database/migrations/2024_12_12_153102_convert_image_credit_fields_in_ifs_to_wysiwyg_field.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+use App\Models\Experience;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use App\Models\ExperienceImage;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('experience_images', function (Blueprint $table) {
+            $table->text('image_credits')->nullable()->after('inline_credits');
+        });
+
+        $experienceImages = ExperienceImage::all();
+
+        foreach ($experienceImages as $instance) {
+            $artist = $instance?->artist;
+            $title = $instance?->credit_title;
+            $date = $instance?->credit_date;
+            $medium = $instance?->medium;
+            $dimensions = $instance?->dimensions;
+            $creditLine = $instance?->credit_line;
+            $referenceNumber = $instance?->main_reference_number;
+            $type = $instance?->credits_input;
+
+            $imageCredit = '';
+
+            $parts = [];
+
+            if ($artist) {
+                $parts[] = "<p>{$artist}</p><br class=\"softbreak\">";
```

Comment:
Are the softbreaks needed here? Wouldn't the <p> provide whatever breaks are needed?

Reply:
For whatever reason they're needed in the wysiwyg editor to break the lines, otherwise they stay on the same line and aren't structured as they should be.